File: app/src/main/java/com/ranzan/mojopizzaclone/MainActivity.java
package com.ranzan.mojopizzaclone;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.ranzan.mojopizzaclone.Fragments.AccountFragment;
import com.ranzan.mojopizzaclone.Fragments.CartFragment;
import com.ranzan.mojopizzaclone.Fragments.ClubFragment;
import com.ranzan.mojopizzaclone.Fragments.FreeFragment;
import com.ranzan.mojopizzaclone.Fragments.Home_Fragment;

public class MainActivity extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener {

    BottomNavigationView bottomNavigation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        bottomNavigation = findViewById(R.id.bottom_navigation);
        bottomNavigation.setOnNavigationItemSelectedListener(this);
        getSupportFragmentManager().beginTransaction().add(R.id.frame, new Home_Fragment()).commit();
    }

    @Override
    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
        switch (item.getItemId()) {
            case R.id.home:
                getSupportFragmentManager().beginTransaction().replace(R.id.frame, new Home_Fragment()).commit();
                return true;
            case R.id.search:
                getSupportFragmentManager().beginTransaction().replace(R.id.frame, new ClubFragment()).commit();
                return true;
            case R.id.cart:
                getSupportFragmentManager().beginTransaction().replace(R.id.frame, new CartFragment()).commit();
                return true;
            case R.id.message:
                getSupportFragmentManager().beginTransaction().replace(R.id.frame, new FreeFragment()).commit();
                return true;
            case R.id.profile:
                getSupportFragmentManager().beginTransaction().replace(R.id.frame, new AccountFragment()).commit();
                return true;
        }
        return false;
    }
}

File: app/src/main/java/com/ranzan/mojopizzaclone/MainActivity.kt
package com.ranzan.mojopizzaclone

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.etebarian.meowbottomnavigation.MeowBottomNavigation
import com.etebarian.meowbottomnavigation.MeowBottomNavigation.ClickListener
import com.etebarian.meowbottomnavigation.MeowBottomNavigation.ShowListener
import com.ranzan.mojopizzaclone.Fragments.*

class MainActivity : AppCompatActivity() {
    private lateinit var bottomNavigation: MeowBottomNavigation
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        bottomNavigation = findViewById(R.id.bottom_navigation)
        bottomNavigation.add(MeowBottomNavigation.Model(1, R.drawable.home))
        bottomNavigation.add(MeowBottomNavigation.Model(2, R.drawable.group))
        bottomNavigation.add(MeowBottomNavigation.Model(3, R.drawable.cart))
        bottomNavigation.add(MeowBottomNavigation.Model(4, R.drawable.offer))
        bottomNavigation.add(MeowBottomNavigation.Model(5, R.drawable.account))
        bottomNavigation.setOnShowListener(ShowListener { item ->
            var fragment: Fragment? = null
            when (item.id) {
                1 -> fragment = Home_Fragment()
                2 -> fragment = ClubFragment()
                3 -> fragment = CartFragment()
                4 -> fragment = FreeFragment()
                5 -> fragment = AccountFragment()
            }
            supportFragmentManager.beginTransaction().replace(R.id.frame, fragment!!).commit()
        })
        bottomNavigation.show(1, true)
        bottomNavigation.setOnClickMenuListener(ClickListener { })
    }
}